examples/csv-payments/self-host/container/demo-client.py

```
@@ -0,0 +1,305 @@
+    last = None
+    interval = 1.0
+    url = f"{args.base_url}/tpf/control-plane/tenants/{args.tenant_id}/executions/{execution_id}"
+    while time.time() < deadline:
+        last = request("GET", url, token=args.control_plane_token, timeout=10)
+        status = last["status"]
+        if status == "SUCCEEDED":
+            print(f"Execution {execution_id} succeeded")
+            return last
+        if status in {"FAILED", "DLQ"}:
+            raise RuntimeError(f"Execution {execution_id} failed: {json.dumps(last, sort_keys=True)}")
+        sleep_for = min(interval + random.uniform(0, 0.25), max(0.0, deadline - time.time()))
```

Ruff (0.15.15)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)